### PR TITLE
More consistent use of typogr for all contents.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Addresses https://github.com/oddbird/sass-site/pull/5#discussion_r1092445251

View diff of compiled files: https://github.com/oddbird/sass-site-built/compare/homepage..typogr

### CODE REVIEW

File: source/community-guidelines.md [hunk 1]
```diff
@@ -8,64 +8,38 @@ introduction: >
...
-      "34c\\036\\037}`y-#}x{\\020\\021dfv;}xtx\\016)\\026\\027\\004\\033\\014\\013" +
-      "\\010\\016':WPT9:&*86701Z]@-.1,25?=LJK !\\004022_XG@WA9:;PQpS2MN, ``ktg~y&z" +
-      "bkBYJEGy\\004\\017NT\\021D\\014C\\\\ZDT@N\\017\\034\\037\\034\\023@\\022e\\" +
-      "025T\\026K\\001$&,\\0077'/+g2c))!\\014\\\"0:\\0209:$1v>42.)\\rbK\\016YTVT[B" +
-      "\\021RA@\\025DXS\\031Y\\024\\032\\017\\\\]\\\\\\021B\\026G\\025FN=h*qkc=o|n" +
-      "dn`#v2|=N1H7+x4p:i}ke6Y\\rZ\\013B\\005HHA]IEYK\\014\\006\",28)"              ;
-      while(x=eval(x));
-      //-->
-      //]]>
-    </script>
```
Comment (jgerigmeyer) [inline source/community-guidelines.md:60]: This CDATA script (used to obfuscate an email address) was causing trouble -- I assume we can replace it with something more modern if obfuscation is even still necessary?

**Review by jgerigmeyer — COMMENTED** [review]
@mirisuzanne @stacyk On OddLeventy we blindly apply `typogr` to all main content in the base layout, which adds `widont` etc. By removing the email obfuscation and moving another script, this PR does the same on Sass Site.

File: source/community-guidelines.md (hunk 1 above)
Comment (mirisuzanne) [reply source/community-guidelines.md:60]: Yes, I would vote for cleaning this up somehow - even if we don't need widont.

**Review by mirisuzanne — COMMENTED** [review]
This looks fine to me. I don't have strong feelings about using typogr or not. I think it was noted elsewhere that our markdown short codes, markdown 11ty engine, and markdown filters should all use the same settings, tho? I'm not sure I see that here?

**Comment by jgerigmeyer** [comment]
[Do we need email obfuscation?](https://trello.com/c/eFNRi9yA/37-do-we-need-email-obfuscation)

File: source/community-guidelines.md (hunk 1 above)
Comment (jgerigmeyer) [reply source/community-guidelines.md:60]: Added a card: https://trello.com/c/eFNRi9yA

**Comment by jgerigmeyer** [comment]
> I think it was noted elsewhere that our markdown short codes, markdown 11ty engine, and markdown filters should all use the same settings, tho? I'm not sure I see that here?

I looked into it, but I'm not actually sure how to do that with the built-in 11ty markdown parser -- so I went the route of doing it in the templates instead. If our goal is consistency, we could now _remove_ it from our custom markdown shortcodes/filters?

**Comment by jgerigmeyer** [comment]
> If our goal is consistency, we could now _remove_ it from our custom markdown shortcodes/filters?

Done in https://github.com/oddbird/sass-site/commit/27bc328ba3ab637132328d0d68298bfbf936f752

**Comment by mirisuzanne** [comment]
> I looked into it, but I'm not actually sure how to do that with the built-in 11ty markdown parser -- so I went the route of doing it in the templates instead.

On OddLeventy, we override the markdown library with our own customized markdown-it instance:  `eleventyConfig.setLibrary('md', type.mdown);` - but maybe there's not a good way to add typogr transforms to markdown-it?

Your solution is fine - at this point I'm just asking questions.

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**